src/main/java/edu/cornell/kfs/vnd/businessobject/VendorDetailExtension.java

```diff
@@ -37,6 +38,9 @@
     protected List<CuVendorCreditCardMerchant> vendorCreditCardMerchants;
     protected String procurementMethods;
 
+    protected boolean paymentWorksOriginatingIndicator;
+    protected Timestamp paymentWorksLastActivityTimestamp;

```

Comment:
The new PR code is populating these two new properties accordingly, but they do not appear to have been added to the UI by this new code. Are these two properties meant to be displayed on the KFS screens and documents? If so, should that setup be done in this PR or in a future PR?

Reply:
Cathy indicated that she wanted these new attributes in place so the source of the vendor data could be tracked and reported on.  She did not specify these attributes should be added to the UI. If Cathy or purchasing decides that these items are needed on the UI, they could be added as we will accurately be capturing the data from the beginning of the PMW implementation.